resource-directory/service/projection.go

```diff
@@ -59,9 +62,13 @@ func (p *Projection) getModels(ctx context.Context, resourceID *commands.Resourc
 	if created {
 		p.cache.Set(resourceID.GetDeviceId(), created, cache.DefaultExpiration)
 	} else {
-		p.cache.Replace(resourceID.GetDeviceId(), created, cache.DefaultExpiration)
+		if err := p.cache.Replace(resourceID.GetDeviceId(), created, cache.DefaultExpiration); err != nil {
```

Comment:
we want to ignore this error - if the resource is not in cache it means that it is stored in persistent cache